### ISSUE TITLE
If I use pre-trained weights, is the output normal?

### ISSUE THREAD

**Issue by jaylongnwnr** [ultralytics/ultralytics#2779]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I thought fine-tuning with the weights already trained he should continue from the lowest point I had last time but it doesn't look like this, am I misunderstanding something or is this normal

**This is the result of the first time, I fine-tuned the second time using the generated weights**
![results1](https://github.com/ultralytics/ultralytics/assets/32053921/5696cd93-1c50-432d-88a1-3e3cd7313199)

**This is the result of the second time**
![results2](https://github.com/ultralytics/ultralytics/assets/32053921/da5265c6-df3d-4b3b-8828-6988ab24580f)

**Comment by Kayzwer** [comment]
make sure you set warm up epoch to 0, as warm up epoch set the learning rate to very high (which ofc reduce the performance at the initial training epoch)

**Comment by jaylongnwnr** [comment]
OK, i will give it a try thank you !

**Comment by HomeNerd93** [comment]
> make sure you set warm up epoch to 0 (when using previous weight), as warm up epoch set the learning rate to very high (which ofc reduce the performance at the initial training epoch)

Please can you demonstrate with an example how to set the warm up epoch? Thanks

**Comment by glenn-jocher** [comment]
@jaylongnwnr certainly! Setting the warm-up epochs to 0 is easy. You can do this by modifying the `hyp.yaml` file located in the `yolov8/data` directory. 

Open the `hyp.yaml` file and locate the line that reads "`warmup_epochs: 3.0`". Change the value to 0 so that is reads "`warmup_epochs: 0`". 

This change will ensure that the learning rate is not set too high for the initial training epoch when using previous weights. I hope you find this helpful, let me know if you have any further questions!

**Comment by jaylongnwnr** [comment]
@glenn-jocher @Kayzwer 

Thank you very much, I have solved my problem.

**Comment by glenn-jocher** [comment]
@HomeNerd93 you're welcome! I'm glad to hear that your issue has been resolved. If you have any further questions or run into any more issues with YOLOv8, feel free to open a new issue and we'll do our best to assist you.
